eventually/authentication/models.py

```diff
@@ -0,0 +1,161 @@
+from django.db import models
+
+# Create your models here.
+
+from django.db import models
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.utils.dateformat import format
+from  datetime import datetime
+
+class CustomUser(AbstractBaseUser):
+    """
+        This class represents a basic user, containing only the essential information about the user.\n
+        1. first, last and middle names: all Char with maximum length of 20 chars.
+        2. email: email field of max length 20 chars
+        3. password: charfield of max length 128 symbols
+        4. updated_at & created_at fields: DateTimeField.
+           created_at can't be changed once created
+    """
+
+    #id - already exists
```

Comment:
We don't need commented line in git repository

Reply:
Deleted.